README.md

```diff
@@ -41,4 +43,59 @@ int main() {
     i2cee.deinit();
 }
 ```
+## Saving struct example
+```cpp
+// Here's an example using a 24LC256
+#include "mbed.h"
+#include "I2CEEBlockDevice.h"
+
+#define BLOCK_SIZE 32
+
+// Create EEPROM device on I2C bus with 32kbytes of memory
+I2CEEBlockDevice i2cee(P0_0, P0_1, 0xA0, 32*1024, BLOCK_SIZE);
+
+uint8_t setting_block_size;
+
+struct t_setting {
+    uint8_t version;
+    char name[20];
+} setting = {
+    1,
+    "Hello World!"
+};
+
+int main() {
+    printf("i2cee struct test\n");
+
+     // No. of bytes to be stored, but topped up to be multiplied by block size
+    unsigned int setting_block_size = ceil(sizeof(setting)/(double)BLOCK_SIZE)*BLOCK_SIZE; 
+
+    // Temporary buffer
+    char buffer[setting_block_size]; 
```

Comment:
You are right, it should be. But as this is a eeprom branch of block memory library and size of eeprom needs to be set by user, it should be fine.